docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -91,6 +91,44 @@
     "import numpy as np"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***Note: The entire XEB workflow can be run by calling `cirq.experiments.parallel_two_qubit_xeb` and the combined single-qubit randomized benchmarking (RB) and XEB workflows can be run by calling `cirq.experiments.run_rb_and_xeb`.***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qubits = cirq.GridQubit.rect(3, 2, 4, 3)\n",
+    "result = cirq.experiments.parallel_two_qubit_xeb(\n",
+    "    sampler=cirq.DensityMatrixSimulator(noise=cirq.depolarize(5e-3)),  # Any simulator or a ProcessorSampler.\n",
+    "    qubits=qubits \n",
+    ")\n",
+    "\n",
+    "# The returned result is an instance of the `TwoQubitXEBResult` class which provides visualization methods like \n",
+    "result.plot_heatmap()  # plot the heatmap of XEB errors\n",
+    "result.plot_fitted_exponential(*qubits[:2])  # plot the fitted model of xeb error of a qubit pair.\n",
+    "result.plot_histogram()  # plot a histogram of all xeb errors.\n",
+    "# And methods to retrieve errors.\n",
+    "result.pauli_error()\n",
+    "result.xeb_error(*qubits[:2])\n",
+    "result.xeb_fidelity(*qubits[:2])"
```

Comment:
This shows only the last value in the NB cell output.  Please convert to print statements to show them all and make it clear which value is which.

Reply:
done